packages/core/src/browser/quick-view-service.ts

```diff
@@ -59,6 +59,14 @@ export class QuickViewService implements QuickOpenModel, QuickOpenHandler, Quick
         });
     }
 
+    hideItem(label: string): void {
+        this.hiddenItemLabels.add(label);
+    }
+
+    showItem(label: string): void {
+        this.hiddenItemLabels.delete(label);
+    }
+
```

Comment:
The set and get do not allow undefined so therefore actually `hiddenItemLabels` will never contain undefined.. so I thought it is cleaner solution.